tests/migrations/test_graph.py

```diff
@@ -151,6 +153,7 @@ def test_circular_graph_2(self):
             graph.forwards_plan, ('C', '0001')
         )
 
+    @expectedFailure
     def test_dfs(self):
```

Comment:
Can you rename the test case (`dfs` doesn't make that much sense anymore in my eyes) and add another non-failing test (i.e. `range(2, 750)` should be fine).